src/data/chargen/books/ninth-world-guidebook

```diff
@@ -0,0 +1,227 @@
+var descriptors = [
+  {
+    name: 'Coraoan',
+    sourcebook: 'Ninth World Guidebook',
+    page: 215,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 215 }
```

Comment:
I can't remember off the top of my head, but I *think* this isn't necessary when it's a duplicate of the main sourcebook/page.